app/lib/efile/md/two_income_subtraction_worksheet.rb

```diff
@@ -45,7 +45,9 @@ def calculate_fed_income(primary_or_spouse)
           .select { |form1099r| form1099r.recipient_ssn == filer_ssn }
           .sum { |form1099r| form1099r.taxable_amount&.round }
 
-        unemployment_income = find_filer_json_for(primary_or_spouse)&.form_1099_gs_total&.round || 0
+        unemployment_income = @direct_file_json_data.form_1099gs
```

Comment:
i think this can be tackled in another story or even just discussed, but it may make sense to create methods for these calculations on the @direct_file_json_data, so we can test the calculations. i know they might not be used elsewhere, but it could be nice to have testable chunks.

i _think_ we discussed this last time, and decided it was OK here. this might just be because i'm looking this over another time.

Reply:
I agree with this instinct, and maybe a future refactor is still a good idea, but I'd like to know for sure how it might get re-used in the future before preemptively moving methods out.